scalableminds/webknossos
# Fix segmentation tab

The segmentation value was always requested for zoomstep 0. If there weren't buckets at the requested location, no value could be found. This PR ensures that the value is requested for the current zoomstep (which mirrors the rendered data)

### Mailable description of changes:
 - Bug fix: Cell IDs are shown in the segmentation tab for all zoomsteps now
- The shortcut to copy the currently hovered cell id (CTRL + I) works in non-volume-tracings, too.

### URL of deployed dev instance (used for testing):
- https://fixsegmentationtab.webknossos.xyz/

### Steps to test:
- hover cells in datasets/tracings with segmentation (view, skeleton trace, volume trace) and use shortcut ctrl + I to copy the id
- open a dataset/tracing without segmentation and ensure that the shortcut doesn't crash (should be a noop)

### Issues:
- fixes #2674 

------
